apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/TestLogger.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using Google.Cloud.Spanner.V1.Logging;
+using Xunit.Abstractions;
+
+namespace Google.Cloud.Spanner.Data.IntegrationTests
+{
+    internal class TestLogger : DefaultLogger
+    {
+        private static Lazy<TestLogger> s_Instance = new Lazy<TestLogger>(
+            () =>
+            {
+                var logger = new TestLogger();
+                return logger;
+            });
+
+        private TestLogger()
+        {
+        }
+
+        public static void Install()
+        {
+            SetLogger(s_Instance.Value);
+        }
+
+        // ReSharper disable once MemberCanBePrivate.Global
+        // ReSharper disable once UnusedAutoPropertyAccessor.Global
```

Comment:
Ok, that makes sense, although I would argue that since the setter is public, the getter should be too. I don't think it should be private. Is the "Unused..." disable necessary?